utils/gtf_to_bed12/gtfTobed.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import os
+# import sys
```

Comment:
Are the commented lines needed?

Reply:
Yeah don't know why I didn't just get rid of them...

Reply:
These lines are removed.